pennylane/pauli/dla/structure_constants.py

```diff
@@ -60,6 +62,7 @@ def structure_constants(
         pauli (bool): Indicates whether it is assumed that :class:`~.PauliSentence` or :class:`~.PauliWord` instances are input.
             This can help with performance to avoid unnecessary conversions to :class:`~pennylane.operation.Operator`
             and vice versa. Default is ``False``.
+        matrix (bool): Whether or not matrix representations are used and output in the structure constants computation. Default is ``False``.
```

Comment:
In the example, we say this only affects the computation under the hood. What is meant by "whether or not matrix representations are used and **output** in the structure constants computation"?

Reply:
Oh yeah that was copy-pasted from lie closure but doesnt make sense here since the output is always a tensor, yeah!